src/prefect/events/clients.py

```diff
@@ -406,8 +406,6 @@ async def _reconnect(self) -> None:
     async def _checkpoint(self, event: Event) -> None:
```

Comment:
We can just drop the `event: Event` argument here, we're not using it to checkpoint anymore.  We should move that debug log to the `_emit`